tools/benchmark.py

```diff
@@ -78,6 +82,22 @@ def main(argv):
             "min": data["min"],
             "max": data["max"]
         }
+
+    # Test only on linux
+    if "linux" in platform:
+        for [name, test_args] in thread_count_tests:
+            filename = test_args[0]
+            strace_output_filename = filename.split(".")[0] + ".out"
```

Comment:
Better to write this to a tmp dir

Reply:
Switched to `NamedTemporaryFile` for this